DependencyInjection/Configuration.php

```diff
@@ -221,6 +221,7 @@ private function getDbalConnectionsNode(): ArrayNodeDefinition
                 ->end()
                 ->booleanNode('disable_type_comments')->end()
                 ->scalarNode('server_version')->end()
+                ->integerNode('idle_connection_ttl')->defaultValue(30)->end()
```

Comment:
pgbouncer [defaults to 600](https://www.pgbouncer.org/config.html#server_idle_timeout) (5 minutes)
```suggestion
                ->integerNode('idle_connection_ttl')->defaultValue(300)->end()
```

Reply:
so 5 min not 1 hour?

Reply:
let's borrow from pgbouncer yes